modules/users/client/components/admin-user-form.component.ts

```diff
@@ -0,0 +1,105 @@
+import {Component, Input, ElementRef, ViewChild, } from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {NgbModal, ModalDismissReasons} from '@ng-bootstrap/ng-bootstrap';
+import { Subject } from 'rxjs/Subject';
+import { RoleService} from '../../../roles/client/services/roles.service';
+import { UserService} from '../../../users/client/services/user.service';
+import { User} from '../../../users/client/models/user.model.client';
+import {NotificationsService} from 'angular2-notifications';
+
+@Component({
+  selector: 'admin-user-modal',
+  templateUrl: './../views/admin-user-form.view.html'
+})
+export class AdminUserForm {
+  closeResult: string;
+  display: boolean;
+  @Input()
+  userSubject:Subject<any>;
+  private user: User;
+  roles: Array<any>
+
+  //this gets the template with the id content to display in the modal
+  @ViewChild('content') content: ElementRef
+
+  constructor(private modalService: NgbModal, 
+    private elRef: ElementRef, 
+    private roleService: RoleService,
+    private userService: UserService,
+    private notificationService: NotificationsService) 
```

Comment:
should change the instance to notificationsService to mirror NotificationsService